### ISSUE TITLE
Define "steps"

### ISSUE THREAD

**Issue by jakearchibald** [whatwg/infra#206]
Some algorithms take a set of steps as a find of "callback", such as "Queue a task". Should this argument type be defined by infra?

**Comment by annevk** [comment]
Yeah, maybe. It's effectively an algorithm so we should maybe make those synonyms or some such?

**Comment by skitzofrenicdev** [comment]
Pull
